mijia-homie/src/mijia-names.rs

```diff
@@ -0,0 +1,115 @@
+//! Example of how to subscribe to readings from one or more sensors.
+
+use eyre::Report;
+use eyre::WrapErr;
+use mijia::bluetooth::MacAddress;
+use mijia::{MijiaSession, SensorProps};
+use std::collections::HashMap;
+use std::fs::OpenOptions;
+use std::io::stdin;
+use std::io::Write;
+use std::str::FromStr;
+use std::time::Duration;
+use tokio::time;
+
+const SCAN_DURATION: Duration = Duration::from_secs(5);
+
+#[tokio::main]
+async fn main() -> Result<(), Report> {
+    pretty_env_logger::init();
+
+    let args: Vec<String> = std::env::args().collect();
+    if args.len() != 2 {
+        eyre::bail!("USAGE: {} /path/to/sensor-names.toml", args[0]);
+    }
+    let sensor_names_filename = args.get(1).map(|s| s.to_owned()).unwrap();
+
+    let excludes = get_known_sensors(&sensor_names_filename)?;
+    println!("ignoring sensors: {:?}", excludes);
+
+    let (_, session) = MijiaSession::new().await?;
+
+    // Start scanning for Bluetooth devices, and wait a while for some to be discovered.
+    session.bt_session.start_discovery().await?;
+    time::sleep(SCAN_DURATION).await;
+
+    // Get the list of sensors which are currently known, connect those which
+    // are not already known about.
```

Comment:
Nit: this comment is confusing, because you are using 'known' to mean two different things.